utils/outputwriter/outputcontent.go

```diff
@@ -605,6 +598,115 @@ func getCveIdsCellData(cveRows []formats.CveRow, issueId string) (ids CellData)
 	return
 }
 
+func getFinalApplicabilityStatus(cves []formats.CveRow) string {
+	if len(cves) == 0 {
+		return ""
+	}
+
+	statuses := []jasutils.ApplicabilityStatus{}
+	for _, cve := range cves {
+		if cve.Applicability != nil && cve.Applicability.Status != "" {
+			status := stringToApplicabilityStatus(cve.Applicability.Status)
+			if status != nil {
+				statuses = append(statuses, *status)
+			}
+		}
+	}
+	if len(statuses) == 0 {
+		return ""
+	}
+	return results.GetFinalApplicabilityStatus(statuses).String()
+}
+
+func stringToApplicabilityStatus(status string) *jasutils.ApplicabilityStatus {
+	switch status {
+	case jasutils.Applicable.String():
+		status := jasutils.Applicable
+		return &status
+	case jasutils.NotApplicable.String():
+		status := jasutils.NotApplicable
+		return &status
+	case jasutils.ApplicabilityUndetermined.String():
+		status := jasutils.ApplicabilityUndetermined
+		return &status
+	case jasutils.MissingContext.String():
+		status := jasutils.MissingContext
+		return &status
+	case jasutils.NotCovered.String():
+		status := jasutils.NotCovered
+		return &status
+	default:
+		return nil
+	}
+}
+
+func getDependencyPathCellData(impactPaths [][]formats.ComponentRow, writer OutputWriter) CellData {
+	if len(impactPaths) == 0 {
+		return NewCellData()
+	}
+
+	// key: "name:version"
+	directDeps := make(map[string]formats.ComponentRow)
+	transitiveDeps := make(map[string]formats.ComponentRow)
+
+	// Extract dependencies from all impact paths
+	for _, path := range impactPaths {
+		if len(path) < 2 {
+			continue
+		}
+		// First element is always a direct dependency
+		first := path[0]
+		key := fmt.Sprintf("%s:%s", first.Name, first.Version)
+		directDeps[key] = first
+
+		for i := 1; i < len(path)-1; i++ {
+			component := path[i]
+			key := fmt.Sprintf("%s:%s", component.Name, component.Version)
+			transitiveDeps[key] = component
+		}
+	}
```

Comment:
The first node in the impact path is the scanned component itself. The second one is the direct dependency, check `getDirectComponentsAndImpactPaths` in security CLI

Reply:
also it means that transitive is: if more than 2 in impact path => the impacted component is transitive